R/code-of-conduct.R

```diff
@@ -7,6 +7,12 @@
 #' <https://contributor-covenant.org>, version 1:
 #' <https://contributor-covenant.org/version/1/0/0>.
 #'
+#' If your package is going to CRAN the link to the CoC in your README must
+#' be an absolute link to a rendered website as `CODE_OF_CONDUCT.md` is not
+#' included in the package sent to CRAN. `use_code_of_conduct()` will
+#' automatically generate this link if you use pkgdown and have set the
+#' `url` field in `pkgdown.yml`; otherwise you'll need to fill it in yourself.
```

Comment:
Maybe we should default to the source of our template and always leave people in a working state?

So: <https://contributor-covenant.org/version/1/0/0>

Reply:
Oh, good idea!